### PR TITLE
AML-2459 & AML 2461

### PR DESCRIPTION
**Pull request by IainStevenson**
Combined PR for Organisations and Agreements decoupling

### CODE REVIEW

File: src/SFA.DAS.EmployerAccounts.Web/Extensions/OrchestratorExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,62 @@
+﻿using System.Threading.Tasks;
+using System.Web;
+using System.Web.Mvc;
+using SFA.DAS.EmployerAccounts.Models.Account;
+using SFA.DAS.EmployerAccounts.Web.Orchestrators;
+using SFA.DAS.EmployerAccounts.Web.ViewModels;
+
+//using SFA.DAS.EAS.Domain.Models.Account;
```
Comment (robinoconnor) [inline src/SFA.DAS.EmployerAccounts.Web/Extensions/OrchestratorExtensions.cs]: is this meant to be comment?

File: src/SFA.DAS.EmployerAccounts.Web/Orchestrators/OrganisationOrchestrator.cs [hunk 2]
```diff
@@ -229,58 +225,58 @@ public virtual void CreateCookieData(HttpContextBase context, EmployerAccountDat
...
-                    Data = new OrganisationDetailsViewModel(),
-                    Status = HttpStatusCode.Conflict
-                };
-
-                return conflictResponse;
-            }
-
-            return null;
-        }
+        //private static string BuildAddressString(GetEmployerInformationResponse response)
```
Comment (robinoconnor) [inline src/SFA.DAS.EmployerAccounts.Web/Orchestrators/OrganisationOrchestrator.cs]: Is this meant to be commented out?

Comment (IainStevenson) [reply src/SFA.DAS.EmployerAccounts.Web/Orchestrators/OrganisationOrchestrator.cs]: Yeah this was left over from Davids changes I sort of left it in to help him out but realized just now its all done now so I will taken it and some other stuff out.

File: src/SFA.DAS.EmployerAccounts.TestCommon/ScenarioCommonSteps/LevyWorkerSteps.cs [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+        {
+            var hashedAccountId = ScenarioContext.Current["HashedAccountId"] as string;
+            var accountId = hashingService.DecodeValue(hashedAccountId);
+            return accountId;
+        }
+
+        private void SetupRefreshLevyMockMessageQueue(IEnumerable<string> payeSchemes, long accountId,
+            CancellationTokenSource cancellationTokenSource)
+        {
+            //var setupSequence = _messageSubscriber.SetupSequence(x => x.ReceiveAsAsync());
```
[the remark this answers is not in the document]
Comment (IainStevenson) [reply src/SFA.DAS.EmployerAccounts.TestCommon/ScenarioCommonSteps/LevyWorkerSteps.cs]: Yes it was

File: src/SFA.DAS.EmployerAccounts.Web/Extensions/OrchestratorExtensions.cs (hunk 1 above)
Comment (IainStevenson) [reply src/SFA.DAS.EmployerAccounts.Web/Extensions/OrchestratorExtensions.cs]: This stuff was a residue from work david did that co-coincided with mine. I did not clear it up properly I am clearing it and one other thing up now.

**Pull request merged**